lang/erlang/Makefile

```diff
@@ -296,26 +296,15 @@ $(call Package/erlang/Default/description)
  This Erlang/OTP package provides functions for exporting XML data to an external format
 endef
 
-# Host
-# host-compile is done with LibreSSL provided by OpenWrt tools/libressl
-
-HOST_CFLAGS += \
-        -DHAS_EVP_PKEY_CTX \
-        -DHAVE_EVP_CIPHER_CTX_COPY
-
 HOST_CONFIGURE_ARGS += \
 	--with-ssl="$(STAGING_DIR_HOST)" \
-	--disable-hipe \
-	--disable-pgo \
-	--disable-smp-support \
 	--without-javac
 
 # Target
 
 CONFIGURE_ARGS += \
-	--disable-hipe \
-	--disable-smp-support \
 	--without-javac \
+	--disable-builtin-zlib \
```

Comment:
This is necessary why?

Reply:
Just to be sure that OpenWrt's zlib will be used, not vendored one. [Buildroot](https://github.com/buildroot/buildroot/blob/b37249581fa4a221923540478db7fa720e04035d/package/erlang/erlang.mk#L76-L77) or [Alpine](https://git.alpinelinux.org/aports/tree/community/erlang/APKBUILD#n73) has similar options.

Reply:
It seems unnecessary indeed - Erlang is getting linked with host zlib  without it. I've removed this line.